mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -58,14 +63,34 @@ const RestorationAimsForm = () => {
   })
   const {
     handleSubmit: validateInputs,
-    formState: { errors }
+    formState: { errors },
+    reset: resetForm
   } = reactHookFormInstance
 
+  const _loadSiteData = useEffect(() => {
+    if (apiAnswersUrl && resetForm) {
+      setIsLoading(true)
+      axios
+        .get(apiAnswersUrl)
+        .then(({ data }) => {
+          setIsLoading(false)
+          const initialValuesForForm = formatApiAnswersForForm({
+            apiAnswers: data,
+            questionMapping: questionMapping.restorationAims
```

Comment:
curious about resetForm ... are we only calling a GET if the form is reset?

Reply:
hmm not sure I understand the `questionMapping` piece. why are we passing that into `resetForm`?  where is it being used afterwards?

Reply:
I think my comment is confusing.  The formatting function could have mapped all the sections all the time, but I opted to invert control to a parameter also to avoid having to flatten the questionMapping object or hardcode what sections we wanted to loop if that makes sense. If not its not important. I am just thinking aloud on a possible variant of the code that I never used which is not important if you are ok with this way. :)